sbt

```diff
@@ -432,22 +432,19 @@ run() {
     addJava "-Djline.terminal=jline.UnixTerminal"
     addJava "-Dsbt.cygwin=true"
   fi
+  
+  local runSbt=(execRunner "$java_cmd" "${java_args[@]}" "${sbt_options[@]}" -jar "$sbt_jar")
 
   if [[ $print_sbt_version ]]; then
-    execRunner "$java_cmd" -jar "$sbt_jar" "sbtVersion" | tail -1 | sed -e 's/\[info\]//g'
+    "${runSbt[@]}" "sbtVersion" | tail -1 | sed -e 's/\[info\]//g'
```

Comment:
This should fix it
```suggestion
    "${runSbt[@]}" -Dsbt.color=never "sbtVersion" | tail -1 | sed -e 's/\[info\]//g'
```

Reply:
Since `-Dsbt.color` is a relatively recent addition, and the runner script needs to work with all the versions of sbt, `-Dsbt.log.noformat=true` might work better.

```suggestion
    "${runSbt[@]}" -Dsbt.log.noformat=true "sbtVersion" | tail -1 | sed -e 's/\[info\]//g'
```